src/data/nat/fib/zeckendorf.lean

```diff
@@ -0,0 +1,229 @@
+/-
+Copyright (c) 2022 Jakub Kądziołka. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakub Kądziołka
+-/
+import data.nat.fib.basic
+import algebra.big_operators.order
+import tactic.omega.main
+import tactic.linarith
```

Comment:
```suggestion
import algebra.big_operators.order
import data.nat.fib.basic
import tactic.omega
```

Reply:
Are you suggesting to import tactic.omega instead of tactic.omega.main? The former makes the linter complain

Reply:
As it should. You are not meant to import `omega`. So far, you were cheating by importing a file which the linter didn't recognize.